package.json

```diff
@@ -31,7 +31,7 @@
     "@kth/log": "^4.0.5"
   },
   "peerdependencies": {
-    "mongoose": "^6.6.0"
+    "mongoose": "^7.0.0"
```

Comment:
Updated mongoose to v7 since it provides types, then we don't need to install @types/mongoose